pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -917,7 +917,7 @@ func (oe *operationExecutor) CheckVolumeExistenceOperation(
 		if attachable != nil {
 			var isNotMount bool
 			var mountCheckErr error
-			if isNotMount, mountCheckErr = mounter.IsLikelyNotMountPoint(mountPath); mountCheckErr != nil {
+			if isNotMount, mountCheckErr = mounter.IsNotMountPoint(mountPath); mountCheckErr != nil {
```

Comment:
I guess for safety, we should use `mounter.IsNotMountPoint` here because it is plugin-agnostic.

A very special case here is `mount.IsNotMountPoint/IsLikelyNotMountPoint` may fail because mount point is broken (e.g. stale NFS file handle). If this happens, reconstruction will fail and never recover because reconstruction happens only once.

I'm not sure if this is expected. If it is not expected, maybe we need to use another way to check volume existence in reconstruction.